portal-web/test/functional/com/liferay/portalweb/macros/CookieBannerAdmin.macro

```diff
@@ -0,0 +1,103 @@
+definition {
+
+	macro configuredToShowBanner {
+		CookieBannerAdmin.gotoSiteSettings();
+
+		CookieBannerAdmin.saveSettings();
+	}
+
+	macro disableDeclineAllButton {
+		AssertClick(
+			locator1 = "CookieBannerAdmin#INCLUDE_DECLINE_ALL_BUTTON",
+			value1 = "Include \&quot;Decline All\&quot; Button");
+
+		Button.clickSave();
+	}
+
+	macro gotoInstanceSettings {
```

Comment:
Extract line 17-line 25 to CookieNavigator.macro

Reply:
Ok